projects/cm_mcu/CommandLineTask.c

```diff
@@ -848,6 +851,72 @@ static int execute(void *p, int argc, char **argv)
   return 0;
 }
 
+static BaseType_t first_mcu_ctl(int argc, char **argv, char *m)
```

Comment:
please move to line 343 (where the prototype is) or put it into one of the files in `commands` directory, maybe `BoardCommands.c`

Reply:
Done.